### DISCUSSION TITLE
GE meeting: Monday May 29?

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#123] [General]
This is Memorial Day, so I am assuming @emilymbender and @danflick won't be joining?

What about others, is someone planning to join? @fcbond @arademaker , anyone else?

**Comment by arademaker** [comment]
I can join.

**Comment by emilymbender** [comment]
Yep, holiday here.

**Comment by fcbond** [comment]
I'm afraid I'm in a wrong time zone so I won't

**Comment by danflick** [comment]
Holiday for me too, sorry.

 Dan

**Comment by olzama** [comment]
Let's cancel this one then and try on June 12!
